ui/raidboss/data/04-sb/ultimate/unending_coil_ultimate.js

```diff
@@ -2394,18 +2394,18 @@
         'Dalamud Dive': '달라가브 강하',
         'Death Sentence': '사형 선고',
         'Deathstorm': '죽음의 폭풍',
-        'Dive . Dynamo/Chariot': 'Dive . Dynamo/Chariot', // FIXME
-        'Dive Dynamo Combo': 'Dive Dynamo Combo', // FIXME
+        'Dive . Dynamo/Chariot': '강하 . 달/강철',
```

Comment:
This is replacing `"Dive + Dynamo/Chariot"` and I think the `.` in the sync could really be a `\\+` (because regular expressions).  It's trying to say "dive followed by either dynamo or chariot", so maybe you want to replace the `.` with `+` or whatever makes sense.

Reply:
It's okay with just using `.`

Reply:
Ah, now I understand. Okay, I'll change it to +